### ISSUE TITLE
Add a dashboard card for next important date

### ISSUE THREAD

**Issue by MysticFragilist** [ApplETS/Notre-Dame#246]
**Is your feature request related to a problem? Please describe.**
As a user, it would be great to have a dashboard card to be able to see the upcoming important dates.

**Describe the solution you'd like**
Create a layout for the card that contains upcoming important dates from ETS. It would be nice to have the card resembling that asymmetrical layout:
![image](https://user-images.githubusercontent.com/25663435/137779966-4f1dd950-88fb-4307-a243-bb4334f31ef5.png)
So we could see not only the first next important date but also the two following date

But nothing is set in stone :smile: The assignee can create any layout he wants.

**Describe alternatives you've considered**

**Additional context**
Linked to #243

**Comment by MysticFragilist** [comment]
blocked by #204
